README.md

```diff
@@ -1241,23 +1244,28 @@ and the Dfinity project. [\[18\]][18]
 The Lightning Network is a proposed message relay network operating at a layer
 above the Bitcoin blockchain, enabling many orders of magnitude improvement in
 transaction throughput by moving the majority of transactions outside of the
-consensus ledger into so-called "payment channels". This is made possible (with
-great difficulty) by the Bitcoin scripting language, which enables parties to
+consensus ledger into so-called "payment channels".
+<!---
+lightning network exists beyond bitcoin
+--->
+Which enables parties to
 enter into stateful contracts where the state can be updated by sharing digital
 signatures, and contracts can be closed by finally publishing evidence onto the
 blockchain, a mechanism first popularized by cross-chain atomic swaps.  By
 openning payment channels with many parties, participants in the Lightning
-Network can become focal points for routing the payments of others, leading to a
-fully connected payment channel network, at the cost of capital being tied up on
-payment channels.
-
-While the Lightning Network can also easily extend across multiple independent
-blockchains to allow for the transfer of _value_ via an exchange market, it
-cannot be used to assymetrically transfer _tokens_ from one blockchain to
-another.  The main benefit of the GnuClear network described here is to enable
-such direct token transfers.  That said, we expect payment channels and the
-Lightning Network to become widely adopted along with our token transfer
-mechanism, for cost-saving and privacy reasons.
+Network can become focal points for routing the payments, and contracts of others, leading to a
+fully connected channel network, at the cost of capital being tied up on
+payment channels. This cost can be completely removed by making it into
```

Comment:
yes, it does sound like GnuClear will have the correct amount of money held in bonds ready to be destroyed. That is one important quality for security, and it indeed protects from consensus forks. 

We also need to worry about soft forks.

An attacker could be interested in destroying as much money in the chain as possible. They could be willing to destroy one of their own money in exchange to destroy 2 moneys on the chain.

How much on-chain value can I destroy in exchange for sacrificing all the ships?
If the on-chain value destroyed is more than 2x as much as all the ships, I am willing to make the trade.

Since the validators can execute any soft fork, they should be able to destroy a lot of value.

I rewrote it again https://github.com/BumblebeeBat/FlyingFox/blob/master/docs/2_types_of_bonds.md